README.md

```diff
@@ -1,23 +1,23 @@
 ![help-activity-icon](images/Activity-help.svg)
 
-Help is an activity which aims to provide users with easy access to documentation and manuals
+Help is an activity which aims to provide users with easy access to documentation and manual.
 
-Help can be accesed
+Help can be accesed.
 * as a [standalone activity](http://activities.sugarlabs.org/en-US/sugar/addon/4051)
 * from [within another activity](#help-from-within-another-activity)
 * at https://help.sugarlabs.org/
 
-### How to edit Help and Contribute:
+### How to edit, Help and Contribute:
```

Comment:
No, in this context "Help" is the name of the activity, so a comma is not correct.